src/sugar3/graphics/popwindow.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (C) 2016 Abhijit Patel
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
```

Comment:
Can we have an example of using this at the top of the file.  This would be more like the documentation standard that we want to achive.

Maybe look at the example from an activity perspective?  For example, I plan to use this API in Bibliography to fix up the modal "new item" winodow.  Read activity will also use it for the bookmark window.